inginious/frontend/plugins/upcoming_tasks/templates/upcoming_task_list.html

```diff
@@ -0,0 +1,53 @@
+{# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
+{# more information about the licensing of this file. #}
+
+{% set registered = user_manager.course_is_user_registered(course) %}
+
+<div id="course" class="section tasks_list card mb-4">
+
+    <div class="content list-group list-group-flush tasks-list">
+        {% for task in tasks %}
+            {% set taskid = task.get_id() %}
+            {% if taskid in tasks_data %}
+                {% set succeeded = tasks_data[taskid]["succeeded"] %}
+                {% set completion = tasks_data[taskid]["grade"] %}
+                <a href="{{ get_homepath() }}/course/{{course.get_id()}}/{{taskid}}" class="list-group-item list-group-item-action
```

Comment:
As I suggested before, the external platform link should be used for LTI exercises as they should be solved from it.

Reply:
I'm not sure about this one. In my opinion, this plugin is useful for exercises available directly from INGInious without using other plateforms. Plateforms using INGInious with integration will not push their students to access this page which is available only within INGInious anyway. I mean I don't see what LTI things have to do with this plugin.
The href is just to redirect the student to the actual task, like it is done on the tasks page of a course.

Don't hesitate to tell me if you see things differently.

Reply:
Therefore LTI course should not be displayed. Either LTI courses are displayed with  the external URL, either there are not. But the usual link won't work for the students if it is a LTI course.

Reply:
I'm not sure how to correctly handle this. 
Do you suggest to filter the courses so that LTI courses are removed from this page?
If I'm not wrong, LTI is just an option to enable in the couse settings... So this would mean, courses with this option enabled would not be shown anymore on the upcoming tasks page...

I don't know exactly how the LTI things work. I mean, aren't they simple courses with tasks in INGInious etc... But with the option to display the task inside an other website...?

Reply:
Damn, you are right, LTI is a bit more complexe than I thought.
I will filter the courses so LTI courses are not displayed on this page.
Anyway student from LTI courses are probably not gonna ever use this plugin.

Reply:
They can use the INGInious platform as well as an external platform. OK if filtered.